### ISSUE TITLE
[HOLD] IOS - Emoji - Emoji sent while offline, showing doubled green lines when back online

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#11202]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Navigate any conversation
2. Disable the internet connection
3. Click on the emoji picker and send emoji
4. Go back online.

## Expected Result:
When switching from offline to online, one emoji must be sent

## Actual Result:
Emoji sent while offline, showing doubled green lines when back online

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- iOS

**Version Number:** 1.2.4.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/191750692-36a68fb7-5e49-447a-80a5-e6597e031e35.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by alex-mechler** [comment]
Does both emojis only show up on the sender's side, or does the recipient also get two copies of the message?

**Comment by kbecciv** [comment]
@alex-mechler  Only appears in sender's side.

**Comment by alex-mechler** [comment]
Okay perfect, this is likely external then, so applying the label!

**Comment by sketchydroide** [comment]
This seems related to https://github.com/Expensify/App/issues/11237
Where a similar thing is happening (it's just one line not 2)  I think this is more about duplication of messages than the Green lines, you can see that it also duplicated the emoji.

**Comment by sketchydroide** [comment]
not sure if that should be am external, as the duplication is normally to do with Sequence numbers, and we are revamping that.
What do you think @Expensify/mobile-deployers?

**Comment by luacmartins** [comment]
>not sure if that should be am external, as the duplication is normally to do with Sequence numbers, and we are revamping that.

I think that makes sense. Can we add these issues related to `sequenceNumbers` to the [tracking issue](https://github.com/Expensify/Expensify/issues/227234) for sequence numbers?

**Comment by sketchydroide** [comment]
> Can we add these issues related to sequenceNumbers to the https://github.com/Expensify/Expensify/issues/227234 for sequence numbers?

Sure, but do you mean add them to the body?

**Comment by luacmartins** [comment]
Sure, we can start a known issues section in the body to keep track of all of them.

**Comment by MitchExpensify** [comment]
[Upwork Job](https://www.upwork.com/jobs/~01319001582edc289e)

**Comment by sketchydroide** [comment]
not overdue

**Comment by sketchydroide** [comment]
will set this as weekly, as there is no need to update this daily

**Comment by MitchExpensify** [comment]
What are we doing with this issue? Is it being handled internally or are we waiting on something before making external?

**Comment by sketchydroide** [comment]
I'm currently trying to figure one out that is similar, was trying to replicate it on Friday locally and wasn't able, I'm going to try this again today and try to use staging API instead of the Dev machine (still a local build though)

Hopefully I will have a better answer soon

**Comment by sketchydroide** [comment]
I can no longer replicate this, neither in staging or dev.
I do get duplicated Messages, but I think that is a different issue and the changes on the sequenceNumber

![Screenshot 2022-10-14 at 16 37 11](https://user-images.githubusercontent.com/242568/195887295-71162aee-e0cb-431a-9f85-650b7b3cb0eb.png)

**Comment by sketchydroide** [comment]
@kbecciv can your team take a look to see if it still happens?

**Comment by roryabraham** [comment]
Oh, I'm 99% certain this is the same bug as "stuck chats", so I'm going to put this on HOLD for the Deprecate Sequence Number doc and take it over.

**Comment by sketchydroide** [comment]
@roryabraham I think the double green lines is no longer happening (the original issue).
The double messages/emojis is the only thing I was able to find this time, but entirely related to the issue, it what now happens instead of the previous bug.
I don't think we need to hold this GH, just make sure the double green lines is no longer happening, is not we can close it.

**Comment by kbecciv** [comment]
@sketchydroide As you mention above - the double green lines is no longer happening (the original issue). Build is 1.2.16.4
The double messages I was able to reproduce this time, please attached.

https://user-images.githubusercontent.com/93399543/196199757-b884d3ec-d21b-4545-b742-31b7dd8d89b8.MP4

**Comment by sketchydroide** [comment]
Thanks, I think the double messages is a different thing, not sure if there is already a GH for it.

**Comment by kbecciv** [comment]
Only KI I think of https://github.com/Expensify/App/issues/10550, please check

**Comment by sketchydroide** [comment]
yep those are the steps I took 👍🏼

**Comment by roryabraham** [comment]
Okay, so we can close this out then if the original issue is not reproducible?

**Comment by MitchExpensify** [comment]
> so we can close this out then if the original issue is not reproducible?

I think so, yes!
